### PR TITLE
BTM-437 refactor store lambda

### PR DESCRIPTION
**Pull request by MelAtGDS**
Re-writing the function that stores cleaned user identity verification events.

### CODE REVIEW

File: src/handlers/store-transactions/types.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    transaction_id?: string;
+  };
+}
+
+export enum Env {
+  STORAGE_BUCKET = "STORAGE_BUCKET",
+  EVENT_DATA_FOLDER = "EVENT_DATA_FOLDER",
+}
+
+export type ConfigCache = ConfigElements.services; // ToDo Needed?
```
Comment (markschnitziusgovuk) [inline src/handlers/store-transactions/types.ts:20]: Address this ToDo?

File: src/handlers/store-transactions/business-logic.test.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      timestamp_formatted: "the formatted timestamp",
+      vendor_id: "some vendor id",
+    };
+  });
+
+  it("returns the key to store the transaction data", async () => {
+    // Arrange
+    const expectedDate = new Date(validIncomingCleanedEventBody.timestamp);
+    const yearMonthDayFolder = formatDate(expectedDate, "/");
+    const expectedKey = `${yearMonthDayFolder}/${validIncomingCleanedEventBody.event_id}.json`;
```
Comment (markschnitziusgovuk) [inline src/handlers/store-transactions/business-logic.test.ts:30]: This basically duplicates the logic in business-logic.ts, so should always pass...  I'd suggest maybe just hardcoding the expected string.

File: src/handlers/store-transactions/types.ts (hunk 1 above)
Comment (MelAtGDS) [reply src/handlers/store-transactions/types.ts:20]: Will do :)

File: src/handlers/store-transactions/business-logic.test.ts (hunk 2 above)
Comment (MelAtGDS) [reply src/handlers/store-transactions/business-logic.test.ts:30]: Good suggestion. This came from the existing tests from the handler which I copied over. Will hardcode timestamp and the expected string.

**Pull request closed without merging**